bundles/org.openhab.binding.innogysmarthome/src/main/resources/ESH-INF/i18n/innogysmarthome_de_DE.properties

```diff
@@ -4,17 +4,17 @@ binding.innogysmarthome.description = Das innogy SmartHome Binding verbindet Dei
 
 # thing types
 thing-type.innogysmarthome.bridge.label = innogy SmartHome Zentrale
-thing-type.innogysmarthome.bridge.description = Der innogy SmartHome Controller (SHC) ist die Zentrale f�r das innogy SmartHome System.
+thing-type.innogysmarthome.bridge.description = Der innogy SmartHome Controller (SHC) ist die Zentrale für das innogy SmartHome System.
```

Comment:
What happend here? Did you change line endings? Where they wrong? or did you editor break the encoding of this file?

Reply:
yep. after commit i saw that the encoding was broken. so i had to fix all "umlauts"

Reply:
What happend is. This file should be encoded `ISO-8859` as per definition of java properties files. You saved it and it was converted to `UTF-8`, which causes the umlauts to become broken. You then 'fixed' the umlauts. However the file is no encoded `UTF-8` instead of `ISO-8859`. Which is why it shows up as a diff here on GitHub.

Did you make any changes to this file? I can't find a change. In which case you should just reset it to the original file.

Reply:
Thx for your explanation. I just changed the auth url. I will fix this.

Reply:
I always had this kind of encoding problems. Now I know why... after all those years :D  You're never too old to learn. Thx again!